im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,426 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Im Gespräch - Hefte der Martin Buber-Gesellschaft (German)</title>
+    <title-short>Im Gespräch</title-short>
+    <id>http://www.zotero.org/styles/im-gespraech-buber-gesellschaft</id>
+    <link href="http://www.zotero.org/styles/im-gespraech-buber-gesellschaft" rel="self"/>
+    <author>
+      <name>Michael Gorki</name>
+      <email>michael.gorki@gmx.de</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="humanities"/>
+    <issn>1616-6094</issn>
+    <updated>2015-09-01T16:08:43+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de">
+    <terms>
+      <term name="accessed">abgerufen am</term>
+      <term name="anonymous">ohne Verfasser</term>
+      <term name="anonymous" form="short">o. V.</term>
+      <term name="no date">ohne Jahresangabe</term>
+      <term name="presented at">präsentiert bei</term>
+      <term name="retrieved">abgerufen am</term>
+      <term name="available at">verfügbar bei</term>
+      <term name="scale">Skala</term>
+      <term name="open-quote">»</term>
+      <term name="close-quote">«</term>
+      <term name="open-inner-quote">›</term>
+      <term name="close-inner-quote">‹</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor" delimiter=": ">
+      <label form="short" text-case="sentence" suffix=" von "/>
+      <name delimiter="/" suffix=". " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" strip-periods="false"/>
+  </macro>
+  <macro name="author">
+    <names variable="author" suffix=": ">
+      <name delimiter="/" delimiter-precedes-last="always" initialize="false" sort-separator=""/>
+      <label form="short" prefix=" (" suffix=")" text-case="capitalize-first"/>
+      <substitute>
+        <names variable="editor" suffix=": "/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" delimiter="/" delimiter-precedes-last="always" et-al-min="4" et-al-use-first="2" initialize-with=". "/>
```

Comment:
couldn't you just set the et al.  in citation and be done with it rather than doing it three times?
Also, sure about the 2? (Smith/Meyer et al. 1999) is certainly possible but seems odd.